gulp/ui-lib.js

```diff
@@ -70,9 +70,13 @@ function copyJsTask() {
 copyJsTask.displayName = taskNamePrefix + 'js';
 copyJsTask.description = 'Copies JS files.';
 
+function copyImgTask() {
+  return gulp.src(`${paths.srcImgDir}/**/*.{gif,jpg,png,svg}`)
```

Comment:
be careful with `paths.srcImgDir`: even if the task per se works OK, these specific images will be also published on npm once we have merged. I think these images should be part of PatternLab directory (configuration file of PatternLab should tell you where the paths for the images should go IIRC)